src/osd/OSD.cc

```diff
@@ -9533,3 +9540,30 @@ void OSD::PeeringWQ::_dequeue(list<PG*> *out) {
   }
   in_use.insert(out->begin(), out->end());
 }
+
+namespace ceph { 
+namespace osd_cmds { 
+
+int heap(CephContext& cct, cmdmap_t& cmdmap, Formatter& f, std::ostream& os)
```

Comment:
The reason I've added them to a named namespace is that I foresee moving these into a new source module. One factor is that this having a function that works in both consoles is a different pattern than already exists in the code; they have to be given some sort of name, and I think the namespace helps to make the purpose clear. As more such commands are added, I'm confident the organizational benefit will provide better returns than for this single function.